docs/superpowers/plans/2026-07-02-openapi-codemode-provider.md

```
@@ -0,0 +1,1009 @@
+        operations.insert(d.operation_id.clone(), OperationHandle {
+            operation_id: d.operation_id, method: d.method, path_template: d.path_template,
+            security: d.security, base_url: base_url.clone(), credential: spec.credential.clone(),
+        });
+    }
+    if converted > MAX_OPERATIONS_PER_SPEC {
+        tracing::warn!(service="openapi", label=%spec.label, kept=MAX_OPERATIONS_PER_SPEC, converted,
+            "openapi: MAX_OPERATIONS_PER_SPEC exceeded — extra operations dropped");
+    }
+    Ok(SpecEntry { operations })
+}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)